katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -0,0 +1,376 @@
+Provides plugin for gofer, which allows communicating with Katello server
+and execute scheduled actions.
+
+%package fact-plugin
+BuildArch:  noarch
+Summary:    Adds an fqdn fact plugin for subscription-manager
+Group:      Development/Languages
+
+Requires:   subscription-manager
+Obsoletes:  katello-agent-fact-plugin <= 3.0.0
+
+%description fact-plugin
+A subscription-manager plugin to add an additional fact 'network.fqdn' if not present
+
+%prep
+%setup -q -n katello-agent-%{version}
+
+%build
+pushd src
+%{__python} setup.py build
+popd
+
+%install
+rm -rf %{buildroot}
+mkdir -p %{buildroot}/%{_sysconfdir}/gofer/plugins
+mkdir -p %{buildroot}/%{_prefix}/lib/gofer/plugins
+
+cp etc/gofer/plugins/katelloplugin.conf %{buildroot}/%{_sysconfdir}/gofer/plugins
+cp src/katello/agent/katelloplugin.py %{buildroot}/%{_prefix}/lib/gofer/plugins
+
+mkdir -p %{buildroot}/%{_prefix}/lib/yum-plugins
+cp src/yum-plugins/package_upload.py %{buildroot}/%{_prefix}/lib/yum-plugins
+cp src/yum-plugins/enabled_repos_upload.py %{buildroot}/%{_prefix}/lib/yum-plugins
+
+mkdir -p %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/
+cp etc/yum/pluginconf.d/package_upload.conf %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/package_upload.conf
+cp etc/yum/pluginconf.d/enabled_repos_upload.conf %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/enabled_repos_upload.conf
+
+mkdir -p %{buildroot}%{_sbindir}
+cp bin/katello-package-upload %{buildroot}%{_sbindir}/katello-package-upload
+cp bin/katello-enabled-repos-upload %{buildroot}%{_sbindir}/katello-enabled-repos-upload
+
+mkdir -p %{buildroot}%{_sysconfdir}/rhsm/pluginconf.d/
+mkdir -p %{buildroot}%{_datadir}/rhsm-plugins/
+cp etc/rhsm/pluginconf.d/fqdn.FactsPlugin.conf %{buildroot}%{_sysconfdir}/rhsm/pluginconf.d/fqdn.FactsPlugin.conf
+cp src/rhsm-plugins/fqdn.py %{buildroot}%{_datadir}/rhsm-plugins/fqdn.py
+
+# cache directory
+mkdir -p %{buildroot}/var/cache/katello-agent/
+
+%if 0%{?fedora} > 18 || 0%{?rhel} > 6
+cp src/yum-plugins/tracer_upload.py %{buildroot}/%{_prefix}/lib/yum-plugins
+cp etc/yum/pluginconf.d/tracer_upload.conf %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/tracer_upload.conf
+cp bin/katello-tracer-upload %{buildroot}%{_sbindir}/katello-tracer-upload
+
+# crontab
+mkdir -p %{buildroot}%{_sysconfdir}/cron.d/
+cp extra/katello-agent-send.cron %{buildroot}%{_sysconfdir}/cron.d/%{name}
+%endif
+
+%clean
+rm -rf %{buildroot}
+
+%post -n katello-agent
+chkconfig goferd on
+%if 0%{?fedora} > 18 || 0%{?rhel} > 6
+  /bin/systemctl start goferd > /dev/null 2>&1 || :
+%else
+  /sbin/service goferd start > /dev/null 2>&1 || :
+%endif
+
+touch /tmp/katello-agent-restart
+exit 0
+
+%posttrans
+katello-package-upload 2> /dev/null
+katello-enabled-repos-upload 2> /dev/null
+exit 0
+
+%postun -n katello-agent
+touch /tmp/katello-agent-restart
+exit 0
+
+%files -n katello-agent
+%defattr(-,root,root,-)
+%config %{_sysconfdir}/gofer/plugins/katelloplugin.conf
+%{_prefix}/lib/gofer/plugins/katelloplugin.*
+
+%doc LICENSE
```

Comment:
Should this be ```%license``` or will that break on older versions?

Reply:
I think it would break RHEL6 and RHEL5.